src/main/java/org/broadinstitute/hellbender/tools/ApplyBQSR.java

```diff
@@ -10,8 +10,10 @@
 import org.broadinstitute.hellbender.engine.FeatureContext;
 import org.broadinstitute.hellbender.engine.ReadWalker;
 import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.exceptions.UserException;
```

Comment:
nope, we were not planning to do this for alpha. we renamed the repos but not packages.